other-docs/guides/upgrading/v3.md

```diff
@@ -55,7 +55,6 @@ The latest CMS Module brings with it WordPress version 5.3. Some of the highligh
 - For super-large images, image resizing now uses a scaled-down version of the original image, significantly improving frontend performance
 - The Block Editor has many accessibility improvements and enhanced layout features
 - Automatic image rotation
```

Comment:
Did we already have automatic image rotation btw?

Reply:
```suggestion
```